config/config.go

```diff
@@ -344,6 +344,24 @@ func (c *Configuration) EnsureFilePermissions() error {
 
 	r := regexp.MustCompile("^[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89ab][a-f0-9]{3}-[a-f0-9]{12}$")
 
+	// add trailing slash on data directory is no trailing slash exists
+	if ! strings.HasSuffix(c.System.Data, "/") {
+		c.System.Data = c.System.Data + "/"
+	}
+
+	// create the daemon-data dir if it doesn't exist
+	p, _ := path.Split(c.System.Data)
+
+	if _, err := os.Stat(c.System.Data); err != nil {
+		// if file doesn't exist
+		if os.IsNotExist(err) {
+			//
+			if err = os.Mkdir(p, 0755); err != nil {
+			}
+			zap.S().Debugf("created %s folder", c.System.Data)
```

Comment:
No need for the debug line here.

Reply:
Habits I have to log so I see where it is in the process. I'll remove it.

Reply:
```suggestion
```